docs/manual/en/introduction/Import-via-modules.html

```diff
@@ -62,11 +62,64 @@ <h2>Importing the module</h2>
 		...
 		</code>
 
-		<h2>Caveats</h2>
-
+		<h2>Importable Examples</h2>
 		<p>
-			Currently it's not possible to import the files within the "examples/js" directory in this way.
-			This is due to some of the files relying on global namespace pollution of THREE. For more information see <a href="https://github.com/mrdoob/three.js/issues/9562" target="_blank">Transform `examples/js` to support modules #9562</a>.
+			The core of three.js is focused on the most important components of a 3D engine. Many other components like loaders or controls are part of the
+			examples directory. three.js ensures to keep these files in sync with the core but users have to import them separately if they are required
```

Comment:
> three.js ensures to keep these files in sync with the core 

This a little incorrect. 

> three.js ensures that these files are kept in sync with the core

Reply:
> but users have to import them separately if they are required

Could we also add a note here saying that users need to make sure they are using the examples that match the three.js version they are using? This might seem obvious, but it seems like this is a common problem.

Reply:
Sorry, guess I hit the "approve" button to early 😅  I can't see any way to undo that.

Reply:
> Could we also add a note here saying that users need to make sure they are using the examples that match the three.js version they are using?

I've tried to address your suggestion with a new section. I hope this makes things more clear.

Reply:
>  I can't see any way to undo that.

It's actually possible to dismiss a review^^.

![image](https://user-images.githubusercontent.com/12612165/56573479-6dfe8100-65c1-11e9-9de5-5c96868f9810.png)

Reply:
Oh, not sure how I missed that! Thanks 😁